packages/scenes/src/components/SceneTimePicker.tsx

```diff
@@ -34,9 +40,21 @@ function SceneTimePickerRenderer({ model }: SceneComponentProps<SceneTimePicker>
       fiscalYearStartMonth={0}
       onMoveBackward={() => {}}
       onMoveForward={() => {}}
-      onZoom={() => {}}
+      onZoom={() => model.onZoom(timeRangeState.value, timeRange.onTimeRangeChange)}
```

Comment:
```suggestion
      onZoom={model.onZoom}
```

No no need to create a new function here on every render :)